presentation/tutorial.jl

```diff
@@ -0,0 +1,815 @@
+### A Pluto.jl notebook ###
+# v0.17.7
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 19b71044-a0c8-468a-8e13-91b17d336497
+import Pkg; Pkg.activate
+
+# ╔═╡ 599711e7-ee51-4ca0-94ac-8c1a67f6ac8d
+using CSV, Microbiome, Microbiome.SparseArrays, BiobakeryUtils, BiobakeryUtils.Conda, Random;
+
+# ╔═╡ 84470294-5a17-469e-890a-5e2b9f777dbb
+using Downloads: download
+
+# ╔═╡ 161527c0-d2c9-4ef8-a223-4946b5f0e083
+# Installing packages
+
+# ╔═╡ e41a1c8a-e51b-475d-8839-83830059c139
+
+
+# BIOBAKERYUTILS DEMO
+
+
+
+# ╔═╡ 22ef03e5-0e90-466d-9718-a104002b280f
+# Starting BiobakeryUtils' metaphlan tutorial
+
+# ╔═╡ 0219658e-222a-403b-af54-0fbeacec8606
+m = metaphlan_profile(joinpath(@__DIR__, "../test/files/metaphlan/SRS014464-Anterior_nares_profile.tsv");  sample="SRS014464")
+
+# ╔═╡ 5a17b7af-a266-4733-bba9-c9e4c29ecfd6
+m["k__Bacteria", "SRS014464"]
+
+# ╔═╡ 6ad723a1-5b19-4ffc-aa7c-1d99449a3e20
+m["o__Pseudomonadales", "SRS014464"] 
+
+# ╔═╡ 83e35adc-1edf-4df6-80af-7b065274c176
+size(m)
```

Comment:
you have a lot `size()` function examples. i feel like after the first one, you don't necessarily need the others?